geode-core/src/main/java/org/apache/geode/pdx/internal/ClientTypeRegistration.java

```diff
@@ -78,6 +78,15 @@ public int defineType(PdxType newType) {
     throw returnCorrectExceptionForFailure(pools, newTypeId, lastException);
   }
 
+  private void copyTypeToOtherPools(PdxType newType, int newTypeId, Pool pool) {
+    try {
+      sendTypeToAllPools(newType, newTypeId, getAllPoolsExcept(pool));
+    } catch (Exception e) {
```

Comment:
I wonder if the new try/catch blocks could be narrowed down to java.net.ConnectException.  I'm not comfortable with this eating all exceptions with only debug logging to tell us that something bad happened.  If there's an NPE we'll want to see it.